apps/bfd-pipeline/bfd-pipeline-rda-grpc/pom.xml

```diff
@@ -0,0 +1,156 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>gov.cms.bfd</groupId>
+        <artifactId>bfd-pipeline-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>bfd-pipeline-rda-grpc</artifactId>
+    <packaging>jar</packaging>
+
+    <description>
+        Loads preadjudicated claims data exported from RDA gRPC service into the BFD database.
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>gov.cms.bfd</groupId>
+            <artifactId>bfd-pipeline-shared-utils</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
I did not try to pare down the dependencies too much.  I retained those that seemed likely to be needed when we have implemented the database updates in later sprints.  Since the extras are retained from the ccw-rif pom.xml file they do not add any extra jars to BFD's deployment requirements.  I think that we should remove extras at the end of this PI though.

Reply:
Yeah as long as we revisit this in the future.  If someone refactors the CCW-RIF and removes a dependency it would suck to still have it referenced here if we didn't need it.